pkg/riskanalysis/cmd.go

```diff
@@ -91,5 +93,16 @@ func (opt *Options) Run() error {
 	}
 	fmt.Fprintf(opt.Out, "Successfully wrote: %s\n", outputFile)
 
+	// Write html file for spyglass
+	riskAnalysisHTMLTemplate := testdata.MustAsset("e2echart/test-risk-analysis.html")
+	title := fmt.Sprintf("Risk Analysis For Job: %s, ID: %d", finalProwJobRun.ProwJob, finalProwJobRun.ID)
```

Comment:
Nit: the spyglass page already has the job name and ID prominently displayed at the top, I'd omit this and just leave it as "Risk Analysis"